### ISSUE TITLE
[ACT] Fix Rule that is incorrectly being triggered: WCAG20_Img_HasAlt

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1127]
This Checker rule 
[WCAG20_Img_HasAlt.ts](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/WCAG20_Img_HasAlt.ts)

is incorrectly being triggered by **_ACT 46ca7f_** [Element marked as decorative is not exposed](https://www.w3.org/WAI/standards-guidelines/act/rules/46ca7f/)
 
**Inapplicable Example 1**
[Open test case](https://w3.org/WAI/content-assets/wcag-act-rules/testcases/46ca7f/a48478825dc5baf21cc79bfcfbe12ed462590f1e.html) 
This img element is not marked as decorative [but, has a valid aria-label instead]. 
`<img src="/test-assets/shared/w3c-logo.png" aria-label="W3C logo" /> `

- [x] Fix rule to not trigger when aria-label is present on img element.

**_WCAG20_Img_HasAlt_**: Images must have an 'alt' attribute with a short text alternative if they convey meaning, or 'alt="" if decorative 

     Violation —  [Image does not have an 'alt' attribute short text alternative](https://unpkg.com/accessibility-checker-engine@3.1.40/help/en-US/WCAG20_Img_HasAlt.html#%7B%22message%22%3A%22Image%20does%20not%20have%20an%20'alt'%20attribute%20short%20text%20alternative%22%2C%22msgArgs%22%3A%5B%22%7B0%7D%22%2C%22%7B1%7D%22%2C%22%7B3%7D%22%2C%22%7B4%7D%22%5D%2C%22value%22%3A%5B%22VIOLATION%22%2C%22FAIL%22%5D%2C%22reasonId%22%3A%22Fail_2%22%7D )
    
Reason ID: **_Fail_2: "Image does not have an 'alt' attribute short text alternative"_**

The ”**_accessible name_**” is all the Checker should care about.  Needs a simple fix to existing rule by using the latest accessible name calculation logic. 

#### Background
See rule and mapping discussion in [ACT 46ca7f - Element marked as decorative is not exposed.docx](https://ibm.ent.box.com/file/1046534624169?s=e7i2yie0flot9isyt66ia34hmr3m6l06) on box

**Comment by philljenkins** [comment]
Help should also be updated to include an aria-label example
[WCAG20_Img_HasAlt.html](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/help-v4/en-US/WCAG20_Img_HasAlt.html)

**Comment by eirckr** [comment]
The WCAG20_img_HasAlt is no logger being trigger per description and test case: 
<img width="769" alt="Screen Shot 2023-01-17 at 3 47 43 PM" src="https://user-images.githubusercontent.com/62436670/213019981-2dff7e9c-ab86-4003-905b-74c3f69e9033.png">
